zk-keygen/src/main.rs

```diff
@@ -176,13 +331,22 @@ impl KeyType {
             KeyType::Aes128 => "aes128.json",
         }
     }
+
+    fn get_key_type(matches: &ArgMatches) -> Self {
```

Comment:
```suggestion
    fn from_matches(matches: &ArgMatches) -> Option<Self> {
```
Kind of nit-picky, but just in case we use this enum elsewhere: the caller should probably decide whether unwrap is appropriate (ie. `type` is definitely present)

Reply:
Bonus points if you implement `std::str::FromStr` on `KeyType` instead

Reply:
Yep, make sense! I implemented `FromStr` for `KeyType` and used the `value_of` function from `clap-v3-utils`.